administrator/components/com_templates/Model/TemplateModel.php

```diff
@@ -10,10 +10,14 @@
 
 defined('_JEXEC') or die;
 
-use Joomla\CMS\Application\ApplicationHelper;
-use Joomla\CMS\Component\ComponentHelper;
```

Comment:
Is this the same as @laoneo wrote regards to \JFactory in the Glip Channel? Should not we do the re-sorting and only make the necessary changes so that we have fewer conflicts?

Reply:
If Anurag is using PHPStorm, then the import sorting is automatically done by the IDE. In the new code it is absolutely fine to use the namespaced classes. It is fine for me.